component/src/main/java/org/wso2/extension/siddhi/io/jms/sink/JMSSink.java

```diff
@@ -104,7 +104,7 @@
                                 + "connection.factory.jndi.name='TopicConnectionFactory'"
                                 + ")" +
```

Comment:
Could we add a new line character("\n") here? Stream definition comes after a the @Sink annotation in the same line.

Reply:
+1 for adding before the streams

Reply:
@RukshiW has fixed this in 2d327ce